hazelcast/src/main/java/com/hazelcast/jet/core/Processor.java

```diff
@@ -159,6 +159,10 @@ default boolean isCooperative() {
     default void init(@Nonnull Outbox outbox, @Nonnull Context context) throws Exception {
     }
 
+    default boolean initIsCooperative() {
+        return true;
```

Comment:
We should default to `false`, for b-w compatibility. Same as in `closeIsCooperative`.